config/rbac/sample.yaml

```diff
@@ -0,0 +1,2 @@
+b:
```

Comment:
still needed?

Reply:
I didn't even realize. Thanks for catching this up.